spec/features/admin_user_create_events_spec.rb

```diff
@@ -24,6 +25,29 @@
         expect(page).to have_content 'New Events'
       end
     end
+
+    let(:event_params) { attributes_for(:event) }
+
+    before do
```

Comment:
We have a separate context for the events new page.
```ruby
context 'on the events new page'
```

Reply:
@alexshgov Just edit and show the page depends by new event page and I'm don't where to put it.

Reply:
Ah, I see. Anyway, here's what structure I expect to see

```ruby
context 'on the events index page' do
  context 'when event exists' do
    it 'has an ability to go to the event edit page'
    it 'has an ability to go to the event show page'
```